### ISSUE TITLE
BaseCommand.option_list deprecated in django 1.10

### ISSUE THREAD

**Issue by rodolphopivetta** [celery/django-celery#457]
```
AttributeError: type object 'BaseCommand' has no attribute 'option_list'
```

ref: https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/#django.core.management.BaseCommand.option_list

**Comment by iXce** [comment]
Duplicate of #447 I think :)

**Comment by rodolphopivetta** [comment]
ops, sorry, I didn't see that issue.

**Comment by akshaysharma096** [comment]
what is the fix ?

**Comment by AveryLiu** [comment]
At the meantime I don't see any workaround except downgrading the django version back to 1.9.2

**Comment by auvipy** [comment]
the fix is to move to argparse!!

**Comment by paul-100** [comment]
is this issue resolved? if not why is it marked closed

**Comment by luisj135** [comment]
hi, fix this problem or still pending?

**Comment by yanikkoval** [comment]
ahh, shame it is not fixed

**Comment by muminoff** [comment]
Seems like they are keeping #447 fixed in master branch (3.2.a1 version). 

For whom, who are in rush like me, I solved this issue with updating my requirements.txt:
```
-e git+https://github.com/celery/django-celery#egg=master
```
